### ISSUE TITLE
FF 3.6.12  - wrong position of dropdown menu (1px left shift)

### ISSUE THREAD

**Issue by andreyvolokitin** [fnagel/jquery-ui#38]
Hello!

I`m currently experiencing a very strange bug with jQuery UI Selectmenu and Firefox. In all another browsers it works fine (I did not check in other versions of Firefox, but it does not matter in this case). Here is description of a problem and steps to reproduce a bug:

1) In the source code go to demos/selectmenu/default.html
2) For the main form declare a fixed width and margin:0 auto (bug will not occure with liquid width or without margin:0 auto)
3) Make a content of a page taller, to make browsers vertical scrollbar appear (bug will not occur without vertical scrollbar)
4) Check out any dropdown menu now ("dropdown" Style will the best example). ul.ui-selectmenu-menu will be wrong positioned, there will be 1px shift to the left

This bug, like I said, appears only in FF, so I`m confused a little bit. Is there any workaround?

**Comment by fnagel** [comment]
I can reproduce this problem in latest official FF but not in FF4 Beta7. Ive checked the bugtracker and found this: http://bugs.jqueryui.com/ticket/6000
Its a truncation problem and occurs only in certain browser window widths. Try it by setting the browser width in one pixel steps!

Perhaps you could help to track down the issue at the bugtracker?

**Comment by KDCinfo** [comment]
I just spent the last 4 hours trying to figure out what was going on with this. Chrome and IE11 were 1px shifted, while FF was perfect. Now that I found this thread, I shifted the Chrome window by 1px, and voila! Fixed. I would think that means the window horiz (x) detection should likely be checked to see if it's odd/even, and if one or the other, push the 'left' dimension up or down. Obviously a little testing would be required, but this drove me absolutely nuts. Hoping either the client is okay with it, or that they have their browser at a perfect width when testing it.
